pytorch/inference/docker/1.12/py3/Dockerfile.graviton.e3.cpu

```diff
@@ -0,0 +1,164 @@
+FROM arm64v8/ubuntu:20.04
```

Comment:
we are now changing this name to "ec2" instead of "e3" - can you change the name of the dockerfile?